app/code/Magento/Checkout/Controller/Cart.php

```diff
@@ -53,13 +59,15 @@ public function __construct(
         \Magento\Checkout\Model\Session $checkoutSession,
         \Magento\Store\Model\StoreManagerInterface $storeManager,
         \Magento\Framework\Data\Form\FormKey\Validator $formKeyValidator,
-        CustomerCart $cart
+        CustomerCart $cart,
+        Escaper $escaper
```

Comment:
Your change is not backward compatible. Could you make it optional with fallback to get this object from object manager?